projects/cashmere/src/lib/app-switcher/app-switcher.module.ts

```diff
@@ -4,22 +4,25 @@ import {HttpClientModule} from '@angular/common/http';
 
 import {AppSwitcherComponent} from './app-switcher.component';
 import {PopModule} from '../pop/popover.module';
+import {ProgressIndicatorsModule} from '../progress-indicators/progress-indicators.module';
 import {AppSwitcherService} from './app-switcher.service';
+import { WorkTrackerService } from '../picklist/services/work-tracker.service';
```

Comment:
...so app switcher is now dependent on picklist?  Seems strange to me...

Reply:
Just a helper service that I didn't realize it was sitting within the picklist directory. I can move it to a more general location if you'd like.

Reply:
Maybe this service should belong someplace else if it is used in multiple places

Reply:
@joeskeen do you think `...\Fabric.Cashmere\projects\cashmere\src\lib\shared` would be an ok place to move `WorkTrackerService`?